### PR TITLE
Create volunteer_moderators.md

### PR DESCRIPTION
**Pull request by jimentwood**
https://app.asana.com/0/1206697630270293/1206706386829518/f

### CODE REVIEW

File: source/about/people/volunteer_moderators.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+## What moderators do
+
+Moderators provide quick checks on submissions. If a submission looks acceptable, the moderator does nothing and the submission gets posted. The vast majority of submissions meet the requirements set by arXiv and can be posted quickly. Only if there is an issue of classification or scholarly content does the moderator need to take action either to change the classification or indicate an issue.
+
+Moderators are not referees and are not doing detailed checks or providing feedback. A typical day may include 10 papers to screen and may take about 10 minutes of time.
+
+## Benefits of moderation are:
+
+- Keeping up to date with newest research in the field
```
Comment (rebamrich) [inline source/about/people/volunteer_moderators.md:13]: To match other bullets: "Keeping up to date with the newest research in the field"

File: source/about/people/volunteer_moderators.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+- Service to your colleagues in support of arXiv’s mission
+- Recognition for that service may count towards professional responsibilities
+- Community - our moderators, advisory board members, and staff are a collegial, engaged, and fun group
+- Involvement with evolving issues of scholarly norms and communications
+
+Moderation of arXiv is a team effort, with multiple moderators in each category working in tandem or trading schedules. This helps ensure that submissions are checked daily while allowing for personal flexibility. Working alongside moderators are the arXiv editorial staff that check submissions for technical requirements and provide support to moderators and authors. As arXiv continues to scale and adapt to community needs, our IT team is researching and developing new tools to supplement moderation.
```
Comment (rebamrich) [inline source/about/people/volunteer_moderators.md:19]: Change to "arXiv editorial staff who check submissions"

File: source/about/people/volunteer_moderators.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
+# Volunteer Moderators
+
+(Content moderation)[/help/moderation/index.html] ensures that arXiv is a valuable resource for researchers. Submissions are checked that they appear in relevant subject areas and are of likely interest to professional researchers. arXiv relies on hundreds of volunteers, who are subject matter experts, to achieve its mission.
```
Comment (rebamrich) [inline source/about/people/volunteer_moderators.md:3]: Second sentence could be more direct and clear:
"arXiv moderators check that submissions appear in relevant..."

**Review by rebamrich — APPROVED** [review]
Looks good! Just a couple suggestions.

File: source/about/people/volunteer_moderators.md [hunk 4]
```diff
@@ -0,0 +1,30 @@
+# Volunteer Moderators
+
+(Content moderation)[/help/moderation/index.html] ensures that arXiv is a valuable resource for researchers. arXiv moderators check that submissions appear in relevant subject areas and are of likely interest to professional researchers. arXiv relies on hundreds of volunteers, who are subject matter experts, to achieve its mission.
+
+## What moderators do
+
+Moderators provide quick checks on submissions. If a submission looks acceptable, the moderator does nothing and the submission gets posted. The vast majority of submissions meet the requirements set by arXiv and can be posted quickly. Only if there is an issue of classification or scholarly content does the moderator need to take action either to change the classification or indicate an issue.
```
Comment (sdorphan) [inline source/about/people/volunteer_moderators.md:7]: I suggest changing "Only if there is an issue of classification or scholarly content" to be more clear.

File: source/about/people/volunteer_moderators.md [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+
+Moderators are not referees and are not doing detailed checks or providing feedback. A typical day may include 10 papers to screen and may take about 10 minutes of time.
```
Comment (sdorphan) [inline source/about/people/volunteer_moderators.md:9]: Change to "A typical day may include screening 10 papers" or "having 10 papers to screen."

File: source/about/people/volunteer_moderators.md [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+## Moderator expectations:
+
+- Check the submission list daily
+- Coordinate schedules with other moderators to ensure daily coverage of submissions
+- Follow arXiv’s evolving editorial code of practice
+- Provide feedback from the research community to your Section Editorial Committee
+
+## Current moderators
+
+All of [arXiv moderators](https://arxiv.org/moderators) are approved by their [Section Editorial Committees](../../about/people/editorial_advisory_council.md#section-editorial-committees) and by arXiv staff.
```
Comment (sdorphan) [inline source/about/people/volunteer_moderators.md:30]: delete "of"

**Review by sdorphan — COMMENTED** [review]
See comments.

**Pull request merged**